RecaptchaValidation/Services/RecaptchaService.cs

```diff
@@ -47,57 +44,51 @@ public async Task<IRecaptchaResponseMessage> Execute()
                 
                 if (!Response.success)
                 {
-                    throw new RecaptchaRequestException();
+                    throw new RecaptchaRequestException($"Errors returned from Recaptcha Verification URL: {Response.error_codes}");
                 }
                 return Response;
             }
             catch (HttpRequestException hre)
             {
                 //Handle Logging Here...
                 Console.WriteLine(hre.ToString());
-                Response = new RecaptchaResponseMessage()
+                return new RecaptchaResponseMessage()
                 {
                     success = false,
                     challenge_ts = DateTime.UtcNow.ToString(),
                     hostname = "localhost", 
                     error_codes = new string[] { $"Error: Status Code {hre.StatusCode} | {hre.Message}" }
                 };
-                return Response;
             }
             catch (RecaptchaRequestException ex)
             {
-
-                /*  Here are the possible "business" level codes:
-                    missing-input-secret    The secret parameter is missing.
-                    invalid-input-secret    The secret parameter is invalid or malformed.
-                    missing-input-response  The response parameter is missing.
-                    invalid-input-response  The response parameter is invalid or malformed.
-                    bad-request             The request is invalid or malformed.
-                    timeout-or-duplicate    The response is no longer valid: either is too old or has been used previously.
-                */
-
                 //Handle Logging Here...
                 Console.WriteLine(ex.ToString());
-                return Response;
+                return new RecaptchaResponseMessage()
+                {
+                    success = false,
+                    challenge_ts = DateTime.UtcNow.ToString(),
```

Comment:
Just know that `DateTime.UtcNow.ToString()` is a violation of Dependency Injection principles.

How do you unit test this services behavior? You will never be able to mock out the result of this call.

Typically you want an interface such as an `ITimeProvider` whose job it is to provide the current time. Then, in unit tests, you are able to mock out the DateTimeOffset that is provided back.

Also, you should prioritize using DateTimeOffset over DateTime. A DateTime will use the current systems clock which might not be the same as the client. You will have difficulty putting a DateTime into context if a user is not from the same time zone as the server's system clock.